documentation/Logical Volume Manager (LVM) format.asciidoc

```diff
@@ -203,7 +205,7 @@ The offset, in bytes, relative from the start of the metadata area
 | 8 | 8 | | Data area size +
 Value in bytes
 | 16 | 4 | | Checksum +
-CRC-32 of [yellow-background]*TODO (metadata?)*
+CRC-32 for the data area
```

Comment:
This seems to be copied from the [NetBSD Ethernet CRC](https://github.com/ChaosData/LVMinfo.py/blob/master/lvminfo.py#L65).